try{
    let a = ["hello world"]
    console.log(a[1])
}catch{
    console.log("error")
}